boards/alientek-pandora/include/board.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2021 Luo Jia (HUST IoT Security Lab)
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_alientek-pandora ALIENTEK-PANDORA
+ * @ingroup     boards
+ * @brief       Support for the Alientek Pandora board
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the Alientek Pandora board
+ *
+ * @author      Luo Jia <luojia@hust.edu.cn>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include <stdint.h>
+
+#include "cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    LED pin definitions and handlers
+ * @{
+ */
+#define LEDR_PIN            GPIO_PIN(PORT_E, 7)
+#define LEDR_MASK           (1 << 7)
+
+#define LEDR_ON             (GPIOB->BSRR = LED0_MASK)
```

Comment:
There are several examples that make use of `LED0_ON`, `LED1_ON` etc if those macros are defined - if you provide those for compatibility those will just work.

Reply:
Yes, I'd do them by #define LED0_ON LEDR_ON and etc.